features/support/pages/app.rb

```diff
@@ -54,6 +54,10 @@ def login_as(user: nil, email: nil, password: nil)
     login_page.submit(email: email, password: password)
   end
 
+  def logout
+    page.click_on(I18n.t("devise.shared.links.sign_out"))
+  end
```

Comment:
Not blocking - We may (later) want to write a wrapper for `page` that overwrites click on and other such things to attempt to find by i18n first; not immediately but it's a thing that I've had to hack around a couple different times now in the app.